apis/r/R/SOMASparseNDArray.R

```diff
@@ -239,11 +244,14 @@ SOMASparseNDArray <- R6::R6Class(
                               arrow::field(nms[3], private$.type))
 
       tbl <- arrow::arrow_table(values, schema = arrsch)
-      spdl::debug("[SOMASparseNDArray::write] array created, writing to {}", self$uri)
+      spdl::debug("[SOMASparseNDArray$write] array created, writing to {} at ({},{})", self$uri,
+                  self$tiledb_timestamp[1],self$tiledb_timestamp[2])
       naap <- nanoarrow::nanoarrow_allocate_array()
       nasp <- nanoarrow::nanoarrow_allocate_schema()
       arrow::as_record_batch(tbl)$export_to_c(naap, nasp)
-      writeArrayFromArrow(self$uri, naap, nasp, "SOMASparseNDArray")
+      writeArrayFromArrow(self$uri, naap, nasp, "SOMASparseNDArray", NULL,
+                          if (is.null(self$tiledb_timestamp[1])) NULL
+                          else c(self$tiledb_timestamp[1], self$tiledb_timestamp[1]))
```

Comment:
> This can be simplified

Yes.  I had that / have that form in other places but believe I did run into trouble in some tests.  The PR, as written, passes its tests.  

Happy to clean that up _if the tests cooperate_.

Reply:
Addressed in #2929